app/views/groups/index.html.erb

```diff
@@ -1,36 +1,55 @@
 <%= render "layouts/sidebar.html.erb" %>
```

Comment:
This is not new to this PR, but I've noticed that sometimes the sidebar covers the rest of the page (and I fix it by using command+minus to make the text smaller). Is this hard to fix?